src/elements/content-uploader/ContentUploader.js

```diff
@@ -62,6 +62,7 @@ import '../common/fonts.scss';
 import '../common/base.scss';
 
 type Props = {
+    allowPrepopulateFiles?: boolean,
```

Comment:
looking at some of the other prop names, what about something like `isPrepopulateFilesEnabled` / `isAutoAddFilesEnabled`?

the reason being that the `is_..._Enabled` format has been used a couple times already: `isFolderUploadEnabled`, `isResumableUploadsEnabled`

so using the existing format would be good since this is also used by third-party customers